ci/get_bundle_requirements.py

```diff
@@ -81,12 +79,7 @@ def get_requirements(bundle, models_path, requirements_file):
             if package_key in metadata.keys():
                 optional_dict = metadata[package_key]
                 for name, version in optional_dict.items():
-                    if name not in SPECIAL_LIB_LIST:
-                        libs.append(f"{name}=={version}")
-                    else:
-                        if "pytorch_version" in metadata.keys():
```

Comment:
What's the reason for this change?

Reply:
when we created the initial maisi bundle, it requires xformers which is not included in MONAI, thus we need to install it when running CI, and xformers cannot be simply installed via pip command, thus we add some other code to handle it